wiki/People/Tournament_Committee/en.md

```diff
@@ -7,7 +7,7 @@ tags:
 
 # Tournament Committee
 
-The **Tournament Committee** is a panel of skilled and established organisers and helpers in the tournament scene. The panel was first announced to the public on July 22, 2021 through the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) in response to the rapidly growing community tournament scene in osu!.
+The **Tournament Committee** is a panel of skilled and established organisers and helpers in the tournament scene. Members of the Tournament Committee are distinguished by their orange user title and [user group](/wiki/People/User_groups) badge that reads `TC`. The panel was first announced to the public on July 22, 2021 through the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) in response to the rapidly growing community tournament scene in osu!.
```

Comment:
like other groups, i think there should be a link to the user group page at the start of the member list

Reply:
Changed the member list on the wiki page in e7dc8920840db2364e5883ed4972ad0ecd8ba00c

Reply:
this should still be applied as-is for consistency with other group wiki pages. can keep your changed in the aforementioned commit too